src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1393,22 +1373,22 @@ public void parseAdminTownCommand(String[] split) throws TownyException {
 			} else if (split[1].equalsIgnoreCase("meta")) {
 
 				checkPermOrThrow(sender, PermissionNodes.TOWNY_COMMAND_TOWNYADMIN_TOWN_META.getNode());
-				handleTownMetaCommand(player, town, split);
+				handleTownMetaCommand(catchConsole(sender), town, split);

```

Comment:
Same with this one, I know it used to require the Player but was there a reason?